server/src/main/java/org/eclipse/openvsx/PublishOptions.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.openvsx;
```

Comment:
Add header comment to new file (see other files).

Reply:
Good spot, thanks!